src/tests/ftest/posix/simul.py

```diff
@@ -0,0 +1,149 @@
+        Test #16: symlink, shared mode.
+        Test #17: readlink, shared mode.
+        Test #18: link to one file, shared mode.
+        Test #19: link to a file per process, shared mode.
+        Test #20: fcntl locking, shared mode.
+        Test #21: open, individual mode.
+        Test #22: close, individual mode.
+        Test #23: file stat, individual mode.
+        Test #24: lseek, individual mode.
+        Test #25: read, individual mode.
+        Test #26: write, individual mode.
+        Test #27: chdir, individual mode.
+        Test #28: directory stat, individual mode.
+        Test #29: statfs, individual mode.
+        Test #30: readdir, individual mode.
+        Test #31: mkdir, individual mode.
+        Test #32: rmdir, individual mode.
+        Test #33: unlink, individual mode.
+        Test #34: rename, individual mode.
+        Test #35: creat, individual mode.
+        Test #36: truncate, individual mode.
+        Test #37: symlink, individual mode.
+        Test #38: readlink, individual mode.
+        Test #39: link to one file, individual mode.
+        Test #40: link to a file per process, individual mode.
+        Test #41: fcntl locking, individual mode.
+    :avocado: recursive
+    """
+
+    def run_simul(self, include=None, exclude=None):
+        """ Run simul
+        include str:  comma-separated list of tests to include
+        exclude str:  comma-separated list of tests to exclude
+        If include value is set, exclude value is ignored and vice versa.
+        """
+        # Assuming all vms use the same OS
+        host_os = distro.linux_distribution()[0].lower()
+        version_os = distro.linux_distribution()[1]
+        dfuse_hosts = self.agent_managers[0].hosts
+
+        if "centos" in host_os:
+            if version_os == "8":
+                simul_dict = {"openmpi": "/usr/lib64/openmpi/bin/simul"}
+            else:
+                simul_dict = {"openmpi": "/usr/lib64/openmpi3/bin/simul"}
+        elif "suse" in host_os:
+            simul_dict = {"openmpi": "/usr/lib64/mpi/gcc/openmpi3/bin/simul"}
+        else:
+            self.log.error(host_os)
+            raise NotImplementedError
+
+        dfuse_mount_dir = self.params.get("mount_dir", '/run/dfuse/*')
+
+        if not self.pool:
+            self.log.info("Create a pool")
+            self.add_pool()
+        if not self.container:
+            self.log.info("Create container")
+            self.add_container(self.pool)
+        self.start_dfuse(dfuse_hosts, self.pool, self.container)
+        self.dfuse.check_running()
+
+        for mpi, simul in simul_dict.items():
+            # The use of MPI here is to run in parallel all simul tests
+            # on a single host.
+            load_mpi(mpi)
+            if include:
+                cmd = "{0} -vv -d {1} -i {2}".format(
+                    simul,
+                    dfuse_mount_dir,
+                    include)
+            else:
```

Comment:
```suggestion
            elif exclude and not include:
```

Reply:
Thank you for the review and comment, 
please see the new commit.